builtin/plugins/user_deployer_whitelist/user_deployer_whitelist.go

```diff
@@ -315,6 +327,9 @@ func (uw *UserDeployerWhitelist) GetDeployedContracts(
 		}
 		return nil, errors.Wrap(err, "Failed to load whitelisted deployers state")
 	}
+	if userDeployer.Inactive {
+		return nil, errors.Wrap(err, "Deployer is inactive")
+	}
```

Comment:
Lets get rid of this, the list of deployed contracts shouldn't change based on the deployer status.

Reply:
done